internal/pkg/service/buffer/worker/distribution/config.go

```diff
@@ -1,31 +1,31 @@
-package node
+package distribution
```

Comment:
Premenoval som `node.Node` na `distribution.Node`, tak bude asi jasnejsie o co sa jedna, ked to v kode niekto uvidi.